packages/orbit-components/src/ListChoice/ListChoice.stories.tsx

```diff
@@ -7,239 +9,185 @@ import RenderInRtl from "../utils/rtl/RenderInRtl";
+    icon: undefined,
+    description: undefined,
+    tabIndex: undefined,
+    selectable: false,
+    onClick: undefined,
+  },
 };
 
-WithAction.args = {
-  title: "Choice Title",
-  description: "Further description",
-  disabled: false,
-  icon: "Accommodation",
-  dataTest: "test",
-};
+export const MultipleChoices: Story = {
+  render: ({ icon, ...args }) => {
+    const Icon = typeof icon === "string" && getIcon(icon);
+
+    return (
+      <>
+        {
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+          Array(...Array(3)).map((_, idx) => (
+            // eslint-disable-next-line react/no-array-index-key
+            <ListChoice key={idx} {...args} icon={<Icon />} />
+          ))
+        }
+      </>
+    );
+  },
 
-WithAction.argTypes = {
-  icon: {
-    options: Object.keys(Icons),
-    control: {
-      type: "select",
+  parameters: {
+    controls: {
+      exclude: ["action", "onClick", "selected"],
     },
   },
 };
 
-export const Playground = ({
-  title,
-  description,
-  selectable,
-  selected,
-  disabled,
-  icon,
-  dataTest,
-}) => {
-  const Icon = getIcon(icon);
-
-  return (
-    <ListChoice
-      title={title}
-      description={description}
-      selectable={selectable}
-      selected={selected}
-      disabled={disabled}
-      icon={Icon && <Icon />}
-      onClick={action("onClick")}
-      dataTest={dataTest}
-    />
-  );
-};
+export const WithAction: Story = {
+  render: ({ icon, ...args }) => {
+    const Icon = typeof icon === "string" && getIcon(icon);
+    const IconButton = typeof icon === "string" && getIcon("Plus");
 
-Playground.story = {
-  parameters: {
-    info: "Some description about this type of InputField in general.",
+    return (
+      <ListChoice
+        {...args}
+        icon={<Icon />}
+        action={
+          <Button
+            onClick={action("onClick")}
+            iconLeft={IconButton && <IconButton />}
+            size="small"
+            type="primarySubtle"
+          />
+        }
+      />
+    );
   },
-};
-
-Playground.args = {
-  title: "Choice Title",
-  description: "Further description",
-  selectable: true,
-  selected: false,
-  disabled: false,
-  icon: "Accommodation",
-  dataTest: "test",
-};
 
-Playground.argTypes = {
-  icon: {
-    options: Object.keys(Icons),
-    control: {
-      type: "select",
+  parameters: {
+    controls: {
+      exclude: ["selected", "selectable", "onClick", "action"],
     },
   },
-};
 
-export const Controlled = ({ icon }) => {
-  const [selected, setSelected] = React.useState(false);
-
-  const Icon = getIcon(icon);
-
-  return (
-    <ListChoice
-      title="Controlled ListChoice"
-      description="This ListChoice is controlled by React.useState"
-      icon={Icon && <Icon />}
-      selectable
-      selected={selected}
-      onClick={e => {
-        action("onClick")(e);
-        setSelected(!selected);
-      }}
-    />
-  );
+  args: {
+    selected: undefined,
```

Comment:
If default is false, this is not necessary here, no? It should work even without it.